### DISCUSSION TITLE
How to add fortify?

### DISCUSSION THREAD

**Discussion by dl-web-dev** [orchidsoftware/platform#1778] [Q&A]
I have installed the fortify package but I am not sure how to actually use the 2FA features in the package. I tried to call $this->twoFactorLayout() but it just says the method was undefined even tho ive added the class. Not sure what im doing wrong, was wondering if there was another example of how to implement.

**Comment by EeKay** [comment]
@tabuna Sorry but I'm feeling kinda noob right now:

I've installed Fortify as you mentioned:

_You have to install Laravel Fortify, and complete all setup without add blades calling in App\Providers\FortifyServiceProvider.php._

I installed the package and published its resources using
`php artisan vendor:publish --provider="Laravel\Fortify\FortifyServiceProvider"`

After that I configured the provider.

To me, it is not clear what to do after:

`composer require orchid/fortify`

I see that some things are added to the vendor folder, but I want to use and customize the orchid/fortify views and get the routing correct. 

Do I need to do this manually or is there any config or Artisan command I can do to get this up and running?

Hope you can help, 

Best regards,

Edwin

**Comment by copostic** [comment]
Same here, it starts to drive me crazy, I tried everything but I cannot manage to make it work... 
I still don't have 2FA options on profile page..
